landscape.yml

```diff
@@ -8034,6 +8072,22 @@ landscape:
             logo: flant-deckhouse.svg
             twitter: https://twitter.com/deckhouseio
             crunchbase: https://www.crunchbase.com/organization/flant
+          - item:
+            name: Flatcar Container Linux
+            description: >-
+              A community Linux distribution designed for container workloads, with high security and low maintenance
+            homepage_url: https://www.flatcar.org/
+            repo_url: https://github.com/flatcar/Flatcar
+            logo: flatcar.svg
+            project: incubating
+            twitter: https://twitter.com/flatcar
+            second_path:
+              - "Wasm / Edge/Bare Metal"
```

Comment:
Can you update `Bare Metal` to `Bare metal` (metal `m` lowercase)? Categories and subcategories names are case sensitive.

Reply:
done :+1: